src/Stripe.net/Services/PaymentSourceService/PaymentSourceCreateOptions.cs

```diff
@@ -0,0 +1,39 @@
+namespace Stripe
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public class PaymentSourceCreateOptions : BaseOptions
+    {
+        /// <summary>
+        /// Either a token, like the ones returned by
+        /// <a href="https://stripe.com/docs/stripe.js">Stripe.js</a>, or a
+        /// <see cref="SourceBankAccount"/> instance containing a user's bank account details.
+        /// </summary>
+        [JsonProperty("bank_account")]
+        [JsonConverter(typeof(AnyOfConverter))]
+        public AnyOf<string, SourceBankAccount> BankAccount { get; set; }
```

Comment:
I don't think we should support `bank_account` and `card`. Those are legacy parameters that were deprecated in 2015-02-18. The details should be passed in `source` instead: https://stripe.com/docs/api/cards/create#create_card-source

Reply:
Fine by me, but both params are still in the spec right now.